ethers-providers/src/rpc/pubsub.rs

```diff
@@ -14,18 +11,6 @@ use std::{
 };
 use tracing::error;
 
-/// A transport implementation supporting pub sub subscriptions.
-pub trait PubsubClient: JsonRpcClient {
```

Comment:
This was mistakenly duplicated and in a glob `::*` re-export so it was never caught.